celery_progress/tasks.py

```diff
@@ -0,0 +1,11 @@
+from celery.signals import task_postrun
+
+
+@task_postrun.connect(retry=True)
+def task_postrun_handler(**kwargs):
+    """Runs after a task has finished. This will update the result backend to include the IGNORED result state.
+
+    Necessary for HTTP to properly receive ignored task event."""
```

Comment:
Does WS get an IGNORED state without this post handler?

Reply:
Yes, the WS receives the IGNORED state from it's own post handler, this is necessary purely to update the state on the backend so that the AsyncResult in the HTTP view can be notified that it's been ignored, as the point of the `IGNORED` state is to not save anything to the backend.